etc/sys_manager.cfg

```diff
@@ -0,0 +1,18 @@
+DEBUG=0
+WRITE_CSV=0
+CSV_FILE=/data/beacon.csv
+SIM_ANT=0
+DEPLOY_DELAY=2700
+RETRY_ANT=2100
+UNMUTE_DELAY=5430
+TIME_HIGH=40
```

Comment:
Verified with CalPoly that TIME_HIGH should be 40 seconds.  In tests on the ground the time was less than 10 seconds, but in a vacuum, the time can be around 20 seconds.  Double this to ensure that the antennae really deploys!